### PR TITLE
Update spam2_controller.rb

### PR DESCRIPTION
**Pull request by JeffyDave**
My FTO issue for Public Lab

I made changes to the issue to serve as my FTO
Screen Shot 2022-10-19 at 12 45 16 PM

Fixes ##11498
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by JeffyDave** [comment]
@TildaDares Please can you review this?

**Comment by JeffyDave** [comment]
@TildaDares I have made the changes but it's still showing that I haven't, is there any way you could help me?

**Comment by JeffyDave** [comment]
@TildaDares I have made the changes, I don't know why a test failed, This is a screenshot of the error
Screen Shot 2022-10-19 at 1 06 18 PM

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @JeffyDave, removing the extra plus sign should fix the error. Thank you!

File: app/controllers/spam2_controller.rb [hunk 1]
```diff
@@ -9,8 +9,8 @@ def _spam
                    @nodes.where(type: 'page', status: 1).order('changed DESC')
                  when 'unmoderated'
                    @nodes.where(status: 4).order('changed DESC')
-                  when 'published'
-                    @nodes.where(status: 1).order('changed DESC')
+                 when 'published'
++                  @nodes.where(status: 1).order('changed DESC')
```
[the remark this answers is not in the document]
Comment (JeffyDave) [reply app/controllers/spam2_controller.rb:13]: Thank you very much.

File: app/controllers/spam2_controller.rb [hunk 2]
```diff
@@ -9,8 +9,8 @@ def _spam
                    @nodes.where(type: 'page', status: 1).order('changed DESC')
                  when 'unmoderated'
                    @nodes.where(status: 4).order('changed DESC')
-                  when 'published'
-                    @nodes.where(status: 1).order('changed DESC')
+                 when 'published'
+                  @nodes.where(status: 1).order('changed DESC')
```
[the remark this answers is not in the document]
Comment (JeffyDave) [reply app/controllers/spam2_controller.rb]: I have made the necessary change.

**Review by TildaDares — APPROVED** [review]
Amazing work @JeffyDave. Thank you!!

**Comment by TildaDares** [comment]
Congratulations on merging your PR @JeffyDave 🎉🎉

**Comment by 7malikk** [comment]
Congratulations @JeffyDave 🍾🎊🎉

**Pull request merged**